### PR TITLE
fix: update help dropdown

### PR DESCRIPTION
**Pull request by 12ian34**
Closes: #2597 

Updates the "join the discussion" to "join the community" with the relevant link.
Adds another dropdown option to create a bug report or feature request.

### CODE REVIEW

File: packages/frontend/src/components/NavBar/HelpMenu/index.tsx [hunk 1]
```diff
@@ -121,14 +121,34 @@ const HelpMenu: FC = () => {
...
                                     <ItemDescription>
                                         Get advice &amp; share best practices
                                         with other users.
                                     </ItemDescription>
                                 </div>
                             </HelpItem>
                         </LinkWrapper>
+                        <LinkWrapper
+                            role="button"
+                            href="Submit a feature request or bug report to improve Lightdash"
```
Comment (rephus) [inline packages/frontend/src/components/NavBar/HelpMenu/index.tsx:134]: This is not a valid URL , URL should be 

for bugs : https://github.com/lightdash/lightdash/issues/new?assignees=&labels=🐛+bug&template=bug_report.md&title= 

for features: https://github.com/lightdash/lightdash/issues/new?assignees=&labels=✨+feature-request&template=feature-request.md&title=

Other links might be available like https://github.com/lightdash/lightdash/issues/new/choose

For some reason, I can't see the changes on Render..

**Review by rephus — CHANGES_REQUESTED** [review]

**Comment by rephus** [comment]
UI looks good 

image

File: packages/frontend/src/components/NavBar/HelpMenu/index.tsx (hunk 1 above)
Comment (12ian34) [reply packages/frontend/src/components/NavBar/HelpMenu/index.tsx]: haha dang. that was a copy paste error on my part! fixed :)

**Review by rephus — APPROVED** [review]
:sheep: it !

**Pull request merged**